src/NPCGeneration/createNPC.ts

```diff
@@ -58,7 +58,7 @@ export const createNPC = (town: Town, base = defaultBase): NPC => {
 
   const firstName = base.firstName || getFirstName(race, base.gender)
   let lastName = base.lastName || getLastName(race)
-  if (lastName === firstName) {
+  if (lastName === '') {
```

Comment:
Not sure that I agree with this; I think that it should be `null`, as it is an intentional absence of data. @Glinkis?

Reply:
Sounds good.